cp-subsystem/fenced-lock-monotonic-fencing-tokens/src/main/java/com/hazelcast/codesamples/cp/fencedlock/MonotonicFencingTokens.java

```diff
@@ -53,7 +53,6 @@ public static void main(String[] args) {
 
         // always destroy CP Subsystem data structures otherwise it can lead to a memory leak
         hz1Lock.destroy();
-        hz2Lock.destroy();
```

Comment:
// The lock switches from the available state to the held state.
        FencedLock hz1Lock = hz1.getCPSubsystem().getLock("my-lock");

        // The lock switches from the available state to the held state.
        FencedLock hz2Lock = hz2.getCPSubsystem().getLock("my-lock");
        
Although we have two lock objects, the underlying reference is to the same lock in the CP Subsystem. 
Do we need to destroy both or only one?

Reply:
If only one - suggest we put a comment next to it, to explain why. I can imagine someone (maybe just me), reading through it quickly and not picking up on that first time.

Thoughts?